elsie/boxmixin.py

```diff
@@ -447,18 +578,38 @@ def draw(ctx):
 
     def code(
         self,
-        language,
-        text,
+        language: str,
+        text: str,
         *,
         tabsize=4,
         line_numbers=False,
         style="code",
         use_styles=False,
         escape_char="~",
         scale_to_fit=False,
-    ):
-        """ Draw a code with syntax highlighting """
-
+    ) -> "BoxMixin":
```

Comment:
This should be "TextBoxItem" because user may call extra methods of this type (e.g. line_box)